components/articles/Diary.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const LEFT_ARROW_KEY = 37;
+const RIGHT_ARROW_KEY = 38;
+
+const Diary = ({ text, author, date, getNextDiary, getPrevDiary }) => (
+  <div className="diary-article tile is-parent">
+    <article className="card tile is-child notification">
+      <h1 className="title">Дзённік дня</h1>
+      <div className="date">
+        <span>{date}</span>
+      </div>
+      <span
+        className="icon is-small arrow arrow-left"
+        role="button"
+        tabIndex="0"
+        onKeyDown={e => e.keyCode === LEFT_ARROW_KEY && getPrevDiary()}
```

Comment:
const handleKeyDown = e => {
}

Reply:
That's good idea, but I don't know what to do with onClick event handling. I could, for example, pass some argument to identify what button was clicked, but I'm not sure that it's really worth doing